File: Programmers/lv1/lv1_핸드폰 번호 가리기.java
https://programmers.co.kr/learn/courses/30/lessons/12948

class Solution {
	public String solution(String phone_number) {
		String answer = "";

		for (int i = 0; i < phone_number.length() - 4; i++) {
			answer += "*";
		}
		answer += phone_number.substring(phone_number.length() - 4);

		return answer;
	}
<<<<<<< HEAD
}
=======
}
>>>>>>> a9d136510c496960db5515689f23f6429eff1525

File: Programmers/C#/lv1/lv1_핸드폰 번호 가리기.cs
https://programmers.co.kr/learn/courses/30/lessons/12948?language=csharp

public class Solution
{
    public string solution(string phone_number)
    {
        string answer = "";

        for (int i = 0; i < phone_number.Length; i++)
        {
            if (i < phone_number.Length - 4)
            {
                answer += "*";
            }
            else
            {
                answer += phone_number[i];
            }
        }
        // 높은 버전에서 string[^4..]: ^ 역순, 4.. 4까지 자르기
        return answer;
    }
}